internal/engine/scheduler.go

```diff
@@ -49,6 +49,19 @@ func (e *Engine) setupJobs() error {
 		return fmt.Errorf("failed to add cleanup job: %w", err)
 	}
 
+	estimateDeletionJobDef := gocron.CronJob("0 */6 * * *", false)
+	if err := e.scheduler.AddSingletonJob(
+		"estimate_deletions",
+		"Estimate Deletions",
+		"Estimates media deletions for reporting",
+		"0 */6 * * *", // Every 6 hours
+		estimateDeletionJobDef,
+		e.runEstimateDeletionsJob,
+		false,
```

Comment:
Is there any downside to setting this to `true` so that the job executes on startup?